### ISSUE TITLE
Panic while getting multiple values when only single value presented by specified key.

### ISSUE THREAD

**Issue by katyo** [AltSysrq/lmdb-zero#11]
I working on db with duplicated values.
I put a single value by specified key and trying to retrieve values using get_multiple/next_multiple.
This gives a panic:

```
... panicked at 'MDB_val ptr is NULL', .../lmdb-zero-0.4.2/src/mdb_vals.rs:34:5
```

It works fine when no values or more than single value present.

This is my code repo (you can simply run `cargo test` to get panic):
```
git clone https://github.com/katyo/dsck -b zero-buggy
```

How I creates database:
https://github.com/katyo/dsck/blob/zero-buggy/src/storage/key2keys.rs#L20

How I puts values:
https://github.com/katyo/dsck/blob/zero-buggy/src/storage/key2keys.rs#L75
(the NODUPDATA flag does not affect to this problem)

How I retieves a values:
https://github.com/katyo/dsck/blob/zero-buggy/src/storage/key2keys.rs#L52

Actual database flags: `DUPSORT | INTEGERKEY | DUPFIXED | INTEGERDUP | CREATE`

**Comment by katyo** [comment]
Hmm... It seems the next solution works for me:

```rust
        match cursor.seek_k::<K, Unaligned<V>>(&access, key) {
            Ok(v) => {
                keyset.insert(v.get());
                loop {
                    match cursor.next_multiple::<[Unaligned<V>]>(&access) {
                        Ok(keys) => for key in keys { keyset.insert(key.get()); },
                        Err(Error::Code(error::NOTFOUND)) => break,
                        Err(e) => return Err(e),
                    };
                }
                Ok(keyset)
            },
            Err(Error::Code(error::NOTFOUND)) => Ok(keyset),
            Err(e) => Err(e),
        }
```

**Comment by AltSysrq** [comment]
@katyo This should be fixed in 0.4.3.

**Comment by katyo** [comment]
Thanks
